shared/api/init.go

```diff
@@ -48,6 +52,21 @@ type InitNetworksProjectPost struct {
 	Project string
 }
 
+// InitStorageVolumesProjectPost represents the fields of a new LXD storage volume along with its associated pool.
+//
+// swagger:model
+//
+// API extension: preseed.
```

Comment:
Just a question, here we are mentioning a `preseed` API extension, but it has not been added?

We already have a `init_preseed` extension, but I think we should add a new one?